tests/gem5/verifier.py

```diff
@@ -91,11 +91,18 @@ def test(self, params):
                             self.test_filename,
                             ignore_regexes=self.ignore_regex,
                             logger=params.log)
+
+        for line in diff:
```

Comment:
I think we probably want to make a subclass of `MatchGoldStandard`. Something like `MatchStats`. This will take as a parameter the stats you want to match (say, `['committedInsts', 'cpu.cycles', ...]`) and check that none of these are in the diff.

BTW, I like the way you're checking the diff. I would have parsed the stats file, but your way is definitely better. :).